source/includes/php-frameworks/symfony/RestaurantController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Document\Restaurant;
+use Doctrine\ODM\MongoDB\DocumentManager;
+use MongoDB\BSON\Regex;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class RestaurantController extends AbstractController
+{
+    private DocumentManager $dm;
+    private LoggerInterface $logger;
```

Comment:
These property names were updated, but various methods below still refer to the underscore-prefixed names.